### PR TITLE
Fixes  #591: Loader size fixed, redundant code removed

### LINKED ISSUE #591: Fix loader size, remove redundant class from Settings.css
**Actual Behaviour**

The loader is way to big for the button. It also uses a custom written class instead of using the material ui raised button. It also contains redundant classes that can be removed by using material ui components instead.

**Expected Behaviour**

- [ ] The code should use standard material ui button.
- [ ] Redundant classes must be removed.

**Screenshots of the issue**

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by AakashMallik**
Fixes #591 

Changes: The changes I have made are as follows:
- [x] The code should use standard material ui button.
- [x] Redundant classes must be removed.

Surge Deployment Link: https://pr-592-fossasia-susi-accounts.surge.sh

Screenshots for the change:

### CODE REVIEW

File: src/components/Settings/Settings.react.js [hunk 1]
```diff
@@ -118,24 +118,29 @@ const AvatarRender = props => {
...
+            style={{
+              margin: '16px 0px',
+              width: '100%',
+              minWidth: '88px',
+              maxWidth: '150px',
+            }}
+            disabled={!props.file}
+            backgroundColor={'#4285f4'}
+            labelColor="#fff"
+            // style={{ width: '275px', margin: '10px 0px' }}
```
Comment (samagragupta) [inline src/components/Settings/Settings.react.js]: Please remove the unnecessary comment.

Comment (AakashMallik) [reply src/components/Settings/Settings.react.js]: @samagragupta Thanks pal, totally missed that. Fixed it. :innocent:

**Comment by AakashMallik** [comment]
@anshumanv I don't follow :sweat_smile:. Moving the field Styles in the buttons would lead to a violation of DRY. I hope I did not misunderstand your intention.

**Comment by anshumanv** [comment]
1. Take the styles object out of the button
2. Group it with other style objects already present.

**Comment by AakashMallik** [comment]
@anshumanv Oh, I get it now. But I guess that would be part of refactoring this file coz there are many such styles lying around waiting to be grouped. So I can start refactoring this file once this PR gets merged. Then I'll address the changes you asked as part of the whole refactoring PR. Sounds okay?

**Pull request merged**